homeassistant/components/sensor/octoprint.py

```diff
@@ -0,0 +1,91 @@
+"""
+Support for monitoring OctoPrint sensors.
+
+Uses OctoPrint REST JSON API to query for monitored variables.
+For more details about this component, please refer to the documentation at
+http://docs.octoprint.org/en/master/api/
+"""
+
+import logging
+import requests
+
+from homeassistant.const import TEMP_CELSIUS, CONF_NAME
+from homeassistant.helpers.entity import Entity
+from homeassistant.loader import get_component
+
+DEPENDENCIES = ["octoprint"]
+
+SENSOR_TYPES = {
+    "actual": ["Current Temperature", TEMP_CELSIUS, "printer"],
+    "text": ["State", None, "printer"],
```

Comment:
These names make very little sense to me. I would prefer it to be more descriptive than `actual`, `text` or `completion`. Actual what? Text of what ?